create-deployment-package.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -euo pipefail
+
+FLATCAR_GCP_DEPLOYMENT_PACKAGE_NAME=${FLATCAR_GCP_DEPLOYMENT_PACKAGE_NAME:-dm_template}
+FLATCAR_GCP_DEPLOYMENT_PACKAGE=${FLATCAR_GCP_DEPLOYMENT_PACKAGE:-${FLATCAR_GCP_DEPLOYMENT_PACKAGE_NAME}.zip}
+FLATCAR_GCP_DEPLOYMENT_PACKAGE_URL=${FLATCAR_GCP_DEPLOYMENT_PACKAGE_URL:-https://mirror.release.flatcar-linux.net/coreos/}
```

Comment:
I would rather add this folder here as subfolder instead of having to maintain that remote zip file. Currently the `REGEX_TMPL` comes out of nowhere if one can't easily read the template it works on.